src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -174,12 +172,16 @@ public void linkLetterHolder(final Long claimId, final String userId) {
         claimRepository.linkLetterHolder(claimId, userId);
     }
 
+    public void linkSealedClaimDocument(final Long claimId, final String documentSelfPath) {
+        claimRepository.linkSealedClaimDocument(claimId, documentSelfPath);
+    }
+
     public void saveCountyCourtJudgment(final long claimId, final CountyCourtJudgment countyCourtJudgment) {
         claimRepository.saveCountyCourtJudgment(claimId, jsonMapper.toJson(countyCourtJudgment));
     }
 
     public void saveDefendantResponse(final long claimId, final String defendantId, final String defendantEmail,
-                               final ResponseData responseData) {
+                                      final ResponseData responseData) {
```

Comment:
I think this tabulation can be removed.

Reply:
There is no tab, just IDE aligned arguments.